_commands/basics/seq.md

```diff
@@ -3,21 +3,93 @@
 
 seq
 -------
+
+'seq' prints out a sequence of numbers to standard out. If no flags are given the seq starts at 1 and increments by 1 until the given number input is reached.
```

Comment:
this shouldn't be a header, also you've got another one line description below this which shows up with the correct formatting. Maybe replace that one (line 9) with this?